extensions/gitlab-docs/src/docs.tsx

```diff
@@ -0,0 +1,162 @@
+import { ActionPanel, Action, List, showToast, Toast } from "@raycast/api";
+import { useState, useEffect, useRef, useCallback } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+const apiUrl =
+  "https://3pncfou757-dsn.algolia.net/1/indexes/*/queries?x-algolia-api-key=89b85ffae982a7f1adeeed4a90bb0ab1&x-algolia-application-id=3PNCFOU757";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search GitLab Docs..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult, index) => (
+          <SearchListItem key={index} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      icon="../assets/docs-icon.png"
+      title={searchResult.name}
+      subtitle={searchResult.description}
+      accessoryTitle={searchResult.category}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <Action.OpenInBrowser title="Show Details" url={searchResult.url} />
```

Comment:
This title is usually used when opening the Detail Panel; maybe use the default title?